src/frontend/src/lib/services/request-poh-credential.services.ts

```diff
@@ -0,0 +1,82 @@
+import {
+	INTERNET_IDENTITY_ORIGIN,
+	POH_ISSUER_CANISTER_ID,
+	POH_ISSUER_ORIGIN
+} from '$lib/constants/app.constants';
+import { i18n } from '$lib/stores/i18n.store';
+import { userProfileStore, type UserProfile } from '$lib/stores/settings.store';
+import { toastsError } from '$lib/stores/toasts.store';
+import { Principal } from '@dfinity/principal';
+import { isNullish, nonNullish } from '@dfinity/utils';
+import {
+	requestVerifiablePresentation,
+	type VerifiablePresentationResponse
+} from '@dfinity/verifiable-credentials/request-verifiable-presentation';
+import { get } from 'svelte/store';
+
+// This credential type is defined by the issuer Decide AI
+const POH_CREDENTIAL_TYPE = 'ProofOfUniqueness';
+
+export const requestPohCredential = async ({
```

Comment:
Nth: We do not have a linter for this but I think we do not have much function that returns a `boolean`. We mostyl use more descriptive results like `Promise<{result?: "ok" | "error", data?: XYZ}>` or `Promise<{success: boolean}`. Particularly given the function is named `requestPohCredential`, it would be a bit more descriptive. WDYT?

Reply:
Makes sense, done!